var/spack/repos/builtin/packages/py-more-itertools/package.py

```diff
@@ -21,6 +22,7 @@ class PyMoreItertools(PythonPackage):
     version('4.1.0',  sha256='c9ce7eccdcb901a2c75d326ea134e0886abfbea5f93e91cc95de9507c0816c44')
     version('2.2',    sha256='93e62e05c7ad3da1a233def6731e8285156701e3419a5fe279017c429ec67ce0')
 
```

Comment:
Can you also add https://github.com/more-itertools/more-itertools/issues/578 as a comment above this line so we rememeber why the version requirement disagrees with the docs?

Reply:
Sure!